scaleapi/__init__.py

```diff
@@ -392,6 +397,9 @@ def get_tasks(
                 If true, returns a pre-signed s3 url for the
                 attachment used to create the task.
 
+            limited_response (bool):
+                If true, returns limited task response.
```

Comment:
probably document it here too, just so people know in the future what it is instead of asking us 😆